src/main/cpp/t04_swap.cpp

```diff
@@ -26,5 +26,28 @@ using namespace std;
 
 int t04_swap() {
     
-    return 0;
-}
+    int n, m, b, c, d;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'b' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4679027)